src/OSPSuite.Presentation/DTO/RenameObjectDTOFactory.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using OSPSuite.Assets;
+using OSPSuite.Core.Domain;
+using OSPSuite.Core.Domain.Services;
+using OSPSuite.Utility.Extensions;
+
+namespace OSPSuite.Presentation.DTO
+{
+   public interface IRenameObjectDTOFactory
+   {
+      RenameObjectDTO CreateFor(IWithName objectBase);
+   }
+
+   public class RenameObjectDTOFactory : IRenameObjectDTOFactory
```

Comment:
That code was mostly in PKSim and some in MoBi. Now we have one place to manae those rules and PKSim can add specific stuff to it